### PR TITLE
feat(Feedback): change icons for O2

### PR DESCRIPTION
**Pull request by RubenMorPov**
[WEB - 443](https://jira.tid.es/browse/WEB-443)

- Updated the success feedback icon for O2 to remove padding.
- Changed error feedback icon for O2.
- Changed info feedback icon for O2.

### CODE REVIEW

**Review by yceballost — CHANGES_REQUESTED** [review]

**Comment by pladaria** [comment]
This icon now has cropped edges 
image

File: src/icons/icon-error.tsx [hunk 1]
```diff
@@ -31,23 +31,30 @@ const IconErrorO2: React.FC = () => {
                 <g className={classes.outerAnimation}>
                     <path
```
[the remark this answers is not in the document]
Comment (RubenMorPov) [reply src/icons/icon-error.tsx:27]: I think is solved now. I used the new ones, but added strokeWith on the exclamation dots. I removed it, so it looks thin. 😄

**Review by yceballost — APPROVED** [review]
perfect!

**Review by pladaria — COMMENTED** [review]
No changes in screenshot tests?

**Comment by atabel** [comment]
> No changes in screenshot tests?

I think we only have screenshots for Movistar

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Pull request merged**